packages/flutter/lib/src/material/data_table.dart

```diff
@@ -150,13 +153,38 @@ class DataRow {
   /// table.
   final List<DataCell> cells;
 
+  /// The color for the row.
```

Comment:
It's probably pedantic to say so (so don't change this line), but this color overrides the default row color. That said, we should document what the default color is, before explaining how to override it.